src/component/container/element-list.tsx

```diff
@@ -208,40 +226,134 @@ export class ElementList extends React.Component {
 		return slotListItem;
 	}
 
+	private handleClick(e: React.MouseEvent<HTMLElement>): void {
+		const element = elementFromTarget(e.target);
+		e.stopPropagation();
+		Store.getInstance().setSelectedElement(element);
+		Store.getInstance().setElementFocussed(true);
+	}
+
+	private handleContextMenu(e: React.MouseEvent<HTMLElement>): void {
+		const element = elementFromTarget(e.target);
+		if (element) {
+			elementMenu(element);
+		}
+	}
+
+	private handleDragEnd(e: React.DragEvent<HTMLElement>): void {
+		this.setState({ dragging: false });
+	}
+
+	private handleDragStart(e: React.DragEvent<HTMLElement>): void {
+		this.setState({ dragging: true });
+		const element = elementFromTarget(e.target);
+
+		if (element) {
+			Store.getInstance().setDraggedElement(element);
+		}
+
+		const target = e.target as HTMLElement;
+		const dragElement = target.querySelector('div');
+
+		// restyle the drag image and move it somewhere invisible
+		if (dragElement) {
+			const dragImg = dragElement.cloneNode(true) as HTMLElement;
+			dragImg.setAttribute('style', DRAG_IMG_STYLE);
+			document.body.appendChild(dragImg);
```

Comment:
I get the idea, but what do mean with "this.dragImg = dragImg;" instead of "e.dataTransfer.setDragImage(dragImg, 75, 15);"? Now it doesn’t set the drag image anymore. Or did you mean to use it in a variable then?

Reply:
Sorry, I emitted the `e.dataTransfer.setDragImage(dragImg, 75, 15);` statement, that's still needed. `this.dragImg` should happen right after `e.dataTransfer.setDragImage(dragImg, 75, 15);`;

Reply:
Solved with 70d4708